lib/screens/shuttle_schedule.dart

```diff
@@ -1,30 +1,249 @@
+  }
+
+  // Builds the table cell to display the departure of the shuttle bus from SGW to Loyola
+  Container buildTableCellSGWtoLoy(String cellDisplay) {
+    return Container(
+      padding: EdgeInsets.all(10.0),
+      child: Center(
+        child: AnimatedDefaultTextStyle(
+          style: isSelectedSGWToLoy
+              ? GoogleFonts.raleway(
+                  fontSize: 20.0,
+                  fontWeight: FontWeight.bold,
+                  color: Colors.black,
+                )
+              : GoogleFonts.raleway(
+                  fontSize: 20.0,
+                  color: Colors.black,
+                ),
+          duration: const Duration(milliseconds: 200),
+          child: Text(
+            cellDisplay,
+          ),
+        ),
+      ),
+    );
+  }
+
+  // Builds the table cell to display the departure of the shuttle bus from Loyola to SGW
+  Container buildTableCellLoyToSGW(String cellDisplay) {
+    return Container(
+      padding: EdgeInsets.all(10.0),
+      child: Center(
+        child: AnimatedDefaultTextStyle(
+          style: isSelectedLoyToSGW
+              ? GoogleFonts.raleway(
+                  fontSize: 20.0,
+                  fontWeight: FontWeight.bold,
+                  color: Colors.black,
+                )
+              : GoogleFonts.raleway(
+                  fontSize: 20.0,
+                  color: Colors.black,
+                ),
+          duration: const Duration(milliseconds: 200),
+          child: Text(
+            cellDisplay,
+          ),
+        ),
+      ),
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(
-          ConcordiaLocalizations.of(context).shuttle,
+    return FutureBuilder(
+        future: getSchedule(),
+        builder: (BuildContext context, AsyncSnapshot snapshot) {
+          if (!snapshot.hasData) {
+            return Center(child: CircularProgressIndicator());
+          } else {
+            List<TableRow> rows = [];
+
+            // note: if schedules between monday, tuesday, wednesday or thursday
+            // change, this will only show that of monday
+            // given they are currently the same
+
+            // iterate through all days
+            for (int k = 0; k < snapshot.data['sgw']['1'].length; k++) {
+              // build the row containing the 4 times
+              TableRow row = TableRow(children: [
+                buildTableCellSGWtoLoy(snapshot.data['sgw']['1'][k]),
+                buildTableCellSGWtoLoy(snapshot.data['sgw']['5'].length > k
+                    ? snapshot.data['sgw']['5'][k] // friday
+                    : '--:--'),
+                buildTableCellLoyToSGW(snapshot.data['loyola']['1'][k]),
+                buildTableCellLoyToSGW(snapshot.data['loyola']['5'].length > k
+                    ? snapshot.data['loyola']['5'][k] // friday
+                    : '--:--'),
```

Comment:
Move to shuttle data as mentioned

Reply:
Good idea, will try that now

Reply:
As mentioned, above: addressed in 46970d74438e8a9a7892b32b8e26c00cf6f03f58